### ISSUE TITLE
Issue with -dbtype option to ANNOVAR annotate_variation.pl

### ISSUE THREAD

**Issue by asoltis** [MathOnco/NeoPredPipe#7]
Hello,

This is a very useful piece of code. I came across an issue related to altering the reference gene database used with ANNOVAR. In the 'usr_paths.ini' file I set the humandb, gene_table, and gene_fasta paramters, but encountered an error during the run. I found the fix for this was to set the -dbtype flag in the call to annotate_variation.pl (annovar_annotation function in vcf_manipulate.py). The current set-up appears to default to refGene models.

As another minor comment - the temporary file directories are created in the local run directory; it could be useful to push these to the output directory with the results.

**Comment by rschenck** [comment]
Thanks for the kind feedback!

We will have to update ANNOVAR's annotate_variation call to be more robust.

I agree with the temporary file directories. We will add this feature along with a number of new features we are currently working on. I will close this once it is pushed. Thanks for using.

**Comment by elakatos** [comment]
We've added support for other gene annotations now: in the most recent version, simply set gene_table (and gene_fasta) and the argument of -dbtype flag is passed on to ANNOVAR's annotate_variation based on the filename. This should work for all annotation databases currently available in ANNOVAR, but let us know if there is an issue with an annotation version, and we'll extend support.

We also changed that now all temporary directories are created inside the output directory, thanks for the suggestion!
